config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'user/show'
```

Comment:
this could be `resources: :users, only: :show`

Reply:
ok, i though about it but I was not sure to do it that way. Now I will make a change